aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -194,9 +194,9 @@ public void testNewInputStreamWithAccessPoint() throws Exception {
   }
 
   @Test
-  @DisabledIf(value = "checkIfS3ExpressHelper",
-          disabledReason = "S3 Express doesn’t support Access Points currently")
   public void testNewInputStreamWithCrossRegionAccessPoint() throws Exception {
+    // S3 Express doesn’t support Access Points currently
+    assumeFalse(S3FileIO.checkIfS3Express(bucketName));
```

Comment:
1. assertJ AssertJ has some really good assumption statements along with its assertions.
2. there's enough commonality that something like

```
requireAccessPointSupport()
```
and use everywhere. That would even work if the test. suite evolved to actually the executable against other S3 implementation and some configuration option was added to especially disable AWS-only tests
```

Reply:
Pushed an update let me know what you think!